handlers/developers_handler.js

```diff
@@ -31,9 +31,9 @@ var DeveloperHandler = {
     })
   },
   /**
-   * Set the response object to be in the format of a successful reponse
-   * @param {object} res - The response object
-   * @param {string} developers - The set of usernames and tokens returned from the query
+   * Set the response Object to be in the format of a successful reponse
+   * @param {Object} res - The response Object
+   * @param {String[]} developers - The set of usernames and tokens returned from the query
```

Comment:
This is still not quite clear. If `developers` is a "set of usernames and tokens" and yet it's an array of strings, then each individual String must be some combination of one username and one token? Are they comma separated or something?